### PR TITLE
Alternate ad-block messages

### PR DESCRIPTION
**Pull request by davidrapson**
Updates ad-block banner to use new Members colours and displays a random version of the message, either daily, weekly, monthly or yearly prices. 

generic
weekly
monthly
yearly

@Calanthe @uplne

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js [hunk 1]
```diff
@@ -20,23 +22,42 @@ define([
     svgs
 ) {
     function init() {
-        var alreadyVisted = storage.local.get('alreadyVisited') || 0,
-            adblockLink = 'https://membership.theguardian.com?INTCMP=adb-mv';
+        var alreadyVisted = storage.local.get('alreadyVisited') || 0;
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js]: Nextgen's JavaScript has a convention of only using a single variable declaration per function.

Comment (davidrapson) [inline static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js]: No problem, perhaps we should add this to the [eslint config](http://eslint.org/docs/rules/one-var.html) then? /cc @OliverJAsh

**Comment by davidrapson** [comment]
@jbreckmckye Any other comments apart from the var?

**Comment by jbreckmckye** [comment]
No, looks like a :+1: to me.

**Comment by davidrapson** [comment]
Confirmed on CODE:

screen shot 2015-09-25 at 11 56 19

**Pull request merged**